python/graphscope/framework/graph.py

```diff
@@ -77,10 +77,6 @@ def add_edges(
     ):
         raise NotImplementedError
 
-    @abstractmethod
-    def unload(self):
-        raise NotImplementedError
```

Comment:
Please keep the `unload()` API, and raising a warning to tell users about the deprecation, like

```
warnings.warn("The Graph.unload() method has been deprecated, please using the `del` operator instead, e.g., `del graph`", DeprecationWarning)
```